Cesium3DTilesSelection/src/TilesetJsonLoader.cpp

```diff
@@ -429,7 +431,7 @@ void parseImplicitTileset(
 
   // create implicit loaders
   uint32_t subtreeLevels = subtreeLevelsIt->value.GetUint();
-  uint32_t availableLevels = availableLevelsIt->value.GetUint();
+  uint32_t availableLevels = availableLevelsIt->value.GetUint() + isMaximumLevel;
```

Comment:
Prefer to use explicit casts:

```suggestion
  uint32_t availableLevels = availableLevelsIt->value.GetUint() + uint32_t(isMaximumLevel);
```